tests/test_sourmash.py

```diff
@@ -33,6 +33,18 @@
 from sourmash_tst_utils import SourmashCommandFailed
 
 
+def test_citation_file():
+    import yaml
+
+    thisdir = os.path.dirname(__file__)
+    citation_file = os.path.join(thisdir, '../CITATION.cff')
+
+    with open(citation_file) as fp:
+        x = yaml.safe_load(fp)
```

Comment:
yaml.safe_load is cool! 
I'm curious what "fp" stands for?

Reply:
`file pointer`, I think. I think it was used a lot in _Programming Python_, or maybe it's just a holdover from my C programming days.